src/routes/console/project-[project]/storage/bucket-[bucket]/create-file/create.svelte

```diff
@@ -47,7 +47,11 @@
             trackEvent(Submit.FileCreate, {
                 customId: !!$createFile.id
             });
+
+            await uploadPromise;
         } catch (e) {
+            uploader.removeAtIndex(0);
```

Comment:
not sure we can should rely too much on the index too much. lets use the file id to remove it from the uploader 👍🏻

Reply:
I don't think we'd have a concrete file ID in a `catch` if the upload fails. Would we?

Reply:
Done.